docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md

```diff
@@ -629,3 +629,105 @@ For example:
 
 Enterprise users may need to [install the Enterprise Traces plugin](/docs/enterprise-traces/latest/setup/setup-get-plugin-grafana/) in their Grafana Enterprise instance to allow configuration of tenants, tokens, and access policies.
 After creating a user and access policy using the plugin, you can configure a data source to point at `http://tempo-enterprise-gateway.tempo-test.svc.cluster.local:3100`.
+
+## Set up metamonitoring
+
+Metamonitoring provides observability for your Tempo deployment by collecting metrics and logs from the Tempo components themselves. This helps you monitor the health and performance of your tracing infrastructure.
+
```

Comment:
If someone is coming into this procedure, are they going to know what the k8s-monitoring configuration refers to? Is this a reference to the k8s Helm chart? We should provide some context here. 

If this is the case, then we should add a sentence or two before the procedure to explain that this section uses a different Helm chart to set up monitoring. Would this be the right doc links? https://grafana.com/docs/grafana-cloud/monitor-infrastructure/kubernetes-monitoring/configuration/helm-chart-config/helm-chart/

For example: 

```suggestion
Setting up metamonitoring for Tempo and GET uses the `kubernetes-monitoring` Helm chart. 
For more information about this Helm chart, refer to [LINK TO DOCUMENTATION].
```